cluster/README.md

```diff
@@ -10,6 +10,12 @@ Bedrock uses three tools to automate cluster deployments that you'll need to ins
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
 - [helm](https://github.com/helm/helm)
 
+Verify that these tools are added to your system's PATH in order to avoid errors during cluster deployment.  (Note to WSL users: You will need to either move the executables to /usr/local/bin, or modify ~/.bashrc to add them to PATH and persist the new additions across restarts.)
+
+In addition, you need the Azure `az` command line tool in order to create and fetch Azure configuration info:
```

Comment:
Let's keep this portion in the cluster/azure subsection since this is strictly only an Azure requirement (if we do a multi-cloud deployment with GCP they do not technically need this for the GCP part)

Reply:
Moved it back.  Great point - I was grouping install tools together and didn't realize that was pulling az cli out of the "Azure" section at first.